webapp/src/utils/readTypedMessages.ts

```diff
@@ -88,9 +88,8 @@ function inspectMessages(node: ts.CallExpression): MessageData[] {
             defaultMessage: argNode.text,
             id: id,
             params:
-              callNode.typeArguments?.map((typeArg) => {
-                const typeNode = typeArg as ts.TypeLiteralNode;
-                const memberNode = typeNode.members[0] as ts.PropertySignature;
+              (callNode.typeArguments?.[0] as ts.TypeLiteralNode)?.members.map((member) => {
```

Comment:
I am not 100% sure that typeARguments[0] is the best approach, but when test the code there wasn't any case where typeArgments.lenght > 1

Reply:
yes
const a = [1, 2, 3]
a[0] // 1
a.[0] // compile error
a?.[0] // 1
a?[0] //compile error

const b = null
b?.[0] // undefinded

Reply:
@richardolsson 👆